### PR TITLE
doc: add section 'References'

### PR DESCRIPTION
**Pull request by umarcor**
# Changes

* doc/conf: add extlink 'ghrepo'.
* doc: add section 'References'.

### CODE REVIEW

File: doc/References.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+References
+##########
+
+* :ghrepo:`olofk/ipyxact`
```
Comment (Paebbels) [inline doc/References.rst:4]: What about these ?
* https://github.com/edaa-org/IPXACT-Schema
* https://pypi.org/project/pyIPXACT/ (renamed to this repository)
* https://pypi.org/project/peakrdl-ipxact/
* https://pypi.org/project/ralbot-ipxact/
* https://pypi.org/project/ipxact2systemverilog/
* https://github.com/iDoka/ipxact-cli-tools

Do we need to reference Kactus2?

Comment (Paebbels) [reply doc/References.rst:4]: No judgement from my side, I just found them and listed them here in the comment.

Comment (umarcor) [reply doc/References.rst:4]: I will add them as well.

We do not *need to* reference Kactus2, but I think we should do it :D I will ad it too.

Comment (umarcor) [reply doc/References.rst:11]: Done.

**Pull request merged**